packages/terra-time-input/tests/wdio/time-input-twelve-hour-mobile-spec.js

```diff
@@ -1,13 +1,19 @@
 /* global browser, Terra, before */
 const viewports = Terra.viewports('medium');
 
+// Color contrast will be resolved in https://github.com/cerner/terra-core/pull/1672
```

Comment:
I don't believe this PR is resolving the color contrast issue

Reply:
One thing you'll learn. Even years of experience can't prevent copy paste errors like this, lol